src/main/java/io/confluent/connect/jdbc/sink/dialect/DbDialect.java

```diff
@@ -43,12 +44,18 @@
 
   private final String escapeStart;
   private final String escapeEnd;
+  private TimeZone timeZone;
```

Comment:
I understand that with these changes the `withTimeZone(...)` method will always be called as the `db.timezone` configuration defaults to UTC. While that should always be the case, do you think it's safer to initialize this to UTC?

Reply:
I thought about initializing UTC as a default, and I decided it's better to get an error than to silently get a value that contradicts what the user configured the connector to do. I will add this comment to the docs for withTimeZone if you agree with my reasoning!

Reply:
Yeah, that's good reasoning.